test/unit/testEligibilityValidationStep.js

```diff
@@ -4,17 +4,36 @@ const {expect} = require('chai');
 const rewire = require('rewire');
 const sinon = require('sinon');
 const EligibilityValidationStep = rewire('app/core/steps/EligibilityValidationStep');
-const schema = require('app/steps/ui/deceased/deathcertificate/schema');
+const schema = require('app/steps/ui/screeners/deathcertificate/schema');
 const steps = {};
 const section = 'deceased';
-const resourcePath = 'deceased/deathcertificate';
+const resourcePath = 'screeners/deathcertificate';
 const i18next = {};
 const pageUrl = '/death-certificate';
 const nextStepUrl = '/deceased-domicile';
 const fieldKey = 'deathCertificate';
 const fieldValue = 'Yes';
 
 describe('EligibilityValidationStep', () => {
+    describe('setFeatureTogglesOnCtx()', () => {
+        it('should set feature toggles in the context', (done) => {
+            let ctx = {};
+            const featureToggles = {
+                isDocumentUploadToggleEnabled: 'document_upload',
+                isIntestacyScreeningToggleEnabled: 'intestacy_screening_questions'
+            };
+
+            const eligibilityValidationStep = new EligibilityValidationStep(steps, section, resourcePath, i18next, schema);
+            ctx = eligibilityValidationStep.setFeatureTogglesOnCtx(ctx, featureToggles);
+
+            expect(ctx).to.deep.equal({
+                isDocumentUploadToggleEnabled: 'document_upload',
+                isIntestacyScreeningToggleEnabled: 'intestacy_screening_questions'
```

Comment:
Can these two toggle values be boolean rather than string values, it'll be more representative of the data being passed in the app.